far/farlang.templ.m4

```diff
@@ -21226,32 +21226,32 @@ MViewSearchFor
 "&Rasti"
 
 MViewSearchForText
-"Искать &текст"
-"Search for &text"
-"Hledat &text"
-"Suchen nach &Text"
-"&Szöveg keresése"
-"Szukaj &tekstu"
-"Buscar cadena de &texto"
-"Hľadať &text"
-"Ricerca Di &Testo"
-"Шукати &текст"
-"Шукаць &тэкст"
-"Ieškoti &teksto"
+"&Текст"
+"&Text"
+"&Text"
+"&Text"
+upd:"&Szöveg"
+upd:"Szukaj &tekstu"
+upd:"Cadena de &texto"
+"&Text"
+"&Testo"
+"&Текст"
+"&Тэкст"
+upd:"Ieškoti &teksto"
 
 MViewSearchForHex
-"Искать 16-ричный &код"
-"Search for &hex"
-"Hledat he&x"
-"Suchen nach &Hex (xx xx …)"
-"&Hexa keresése"
-"Szukaj &wartości szesnastkowych"
-"Buscar cadena &hexadecimal"
-"Hľadať he&x"
-"Ricerca &Esadecimale"
-"Шукати 16-ричний &код"
-"Шука&ць hex"
-"Ieškoti &hex"
+"16-ричный &код"
+"&Hex"
+"He&x"
+"&Hex (xx xx …)"
+upd:"&Hexákra"
+upd:"Szukaj &wartości szesnastkowych"
```

Comment:
@BestiaPL, ... and here.

Reply:
S&zesnastkowo

(I assume the text will fit in the window).

Reply:
Yes, it will. Thank you!